packages/rps/src/utils/channelClient.ts

```diff
@@ -14,6 +16,14 @@ export type MessageNotificationName = 'MessageQueued';
 
 export type NotificationName = ChannelNotificationName | MessageNotificationName;
 
+export enum ErrorCodes {
+  SIGNING_ADDRESS_NOT_FOUND = -32100,
+  INVALID_APP_DEFINITION = -32101,
+  INVALID_APP_DATA = -32102,
+  UNSUPPORTED_TOKEN = -32103,
+  CHANNEL_NOT_FOUND = -32104,

```

Comment:
I don't think you're allowed to use these values for application errors. From [the spec](https://www.jsonrpc.org/specification):

> The error codes from and including -32768 to -32000 are reserved for pre-defined errors.

> The remainder of the space is available for application defined errors.